recipes/rnabob/removeman.patch

```diff
@@ -0,0 +1,10 @@
+--- Makefile	2016-09-18 01:24:06.664551971 +0200
++++ Makefile-new	2016-09-18 02:00:43.872926057 +0200
+@@ -58,7 +58,6 @@
+ 
+ install: $(PROGS)
+ 	cp $(PROGS)   $(BINDIR)/
+-	cp rnabob.man $(MANDIR)/man1/rnabob.1
```

Comment:
That's completely fine. You could also use `cp rnabob.man $(BINDIR)/../share/man/rnabob.1`. Assuming BINDIR is `$PREFIX/bin` than the man page should end up under $PREFIX/share/man/rnabob.1 correct?